[2024-11-01T19:37:43.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-01T19:37:43.940+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.add_user_reviews manual__2024-11-01T19:36:55.392647+00:00 [queued]>
[2024-11-01T19:37:43.945+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.add_user_reviews manual__2024-11-01T19:36:55.392647+00:00 [queued]>
[2024-11-01T19:37:43.946+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-01T19:37:43.951+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): add_user_reviews> on 2024-11-01 19:36:55.392647+00:00
[2024-11-01T19:37:43.954+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=180) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-01T19:37:43.955+0000] {standard_task_runner.py:72} INFO - Started process 184 to run task
[2024-11-01T19:37:43.956+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'add_user_reviews', 'manual__2024-11-01T19:36:55.392647+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpuk6y_wyj']
[2024-11-01T19:37:43.958+0000] {standard_task_runner.py:105} INFO - Job 219: Subtask add_user_reviews
[2024-11-01T19:37:43.979+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.add_user_reviews manual__2024-11-01T19:36:55.392647+00:00 [running]> on host eb7019e3fc71
[2024-11-01T19:37:44.026+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='add_user_reviews' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T19:36:55.392647+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-01T19:36:55.392647+00:00'
[2024-11-01T19:37:44.026+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-01T19:37:58.018+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-01T19:37:58.038+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-01T19:37:58.039+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=add_user_reviews, run_id=manual__2024-11-01T19:36:55.392647+00:00, execution_date=20241101T193655, start_date=20241101T193743, end_date=20241101T193758
[2024-11-01T19:37:58.057+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-01T19:37:58.071+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-01T19:37:58.072+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
